src/lib/homestore_backend/resync_blob_data.fbs

```diff
@@ -0,0 +1,14 @@
+namespace homeobject;
+
+table BlobData {
+    shard_id : uint64;
+    key : [ubyte];
```

Comment:
Is this the user_key ? Then can rename it s/key/user_key/

Reply:
done